README.md

```diff
@@ -1,84 +1,69 @@
+### Day 1 - Provision RHOCP on bare metal cluster 
+This project mainly focuses on bare metal OCP cluster. [Distributed CI](https://doc.distributed-ci.io/dci-openshift-agent/) is used to provision the cluster. Users can also refer to [bare metal OCP cluster installation instructions](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-overview.html) to provision bare metal OCP cluster.
 
-To avoid rebooting the nodes and some other issues on Day 2, Some Machine Configurations operations can be enforced on day 1 when provisioning the cluster. The related discussion is ongoing.
+**Note:** To avoid rebooting the nodes and prevent from other potential issues about cluster on Day 2, Machine Configuration operations can be enforced on day 1 while provisioning the cluster. **Discussions regarding this issue are ongoing**.
 
 ### Day 2.0 - Zero O’clock of Day 2
-The Day 2.0 concept is introduced for users to provision intel hardware features right after provisioning an OCP cluster and before any user workloads are deployed. Refer to the steps below to provision Intel hardware features:
-
-* **[Deploy Node Feature Descovery on OpenShift](nfd/README.md#steps-to-install-and-configure-nfd-operator-on-ocp-cluster)**
+The Day 2.0 concept is introduced for users to provision Intel hardware features right after provisioning an OCP cluster and before any user workloads are deployed. Refer to the steps below to provision Intel hardware features:
+* **[Deploy Node Feature Discovery on OpenShift](nfd/README.md#steps-to-install-and-configure-nfd-operator-on-ocp-cluster)**
 * **[Setup Machine Configuration on OpenShift](machine_configuration/README.md#general-configuration-for-provisioning-intel-hardware-features)**  
 
-Note: Running the above steps on Day 2.0 is recommended. However, if you want to provision the features above with the existing cluster on day 2, please be advised that some machine configuration operations might trigger the pods to drain and reboot the nodes. Some of the ongoing efforts in the MCO upstream are to set the machine configurations without rebooting.
+**Note:** Running the above steps on Day 2.0 is recommended. However, if you want to provision the features above with the existing cluster on day 2, please be advised that some machine configuration operations might trigger the pods to drain and reboot the nodes. Some of the ongoing efforts in the MCO upstream are to set the machine configurations without rebooting.
 
-### Day 2 - OCP platform is installed and ready to begin providing services.
+### Day 2 - Red Hat OpenShift cluster is up and running
 Multiple operators are used to provision Intel hardware features and deploy, manage the e2e solutions, and reference workloads.
 
 **Provisioning Intel Hardware features on OpenShift**
```

Comment:
Please split the following PR to another PR.

Reply:
Sounds good. Updated PR and removed this.. Will create another separate PR to add this.